target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -149,6 +149,9 @@ get_status_led() {
 	linkits7688d)
 		[ "$1" = "upgrade" ] && status_led="mediatek:orange:wifi"
 		;;
+	maestro-e220)
+		status_led="$board:red:alert"
```

Comment:
Please use green:power here. I guess we shouldn't scare users with a red alert led as long as everything is fine.